fftool

```diff
@@ -2246,9 +2394,8 @@ class system(object):
 
         ET.indent(root, space=' ')
         ftree.write('field.xml')
-        
```

Comment:
So you think we should suppress the ability to handle pdf files with large-ish systems. Shouldn't we leave this for vmd for example?